<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SEProteinConfigStatsAppGUIClass</class>
 <widget class="QWidget" name="SEProteinConfigStatsAppGUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>251</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Number of residues</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_NR">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Step</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_S">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Reading path</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_r"/>
     </item>
     <item>
      <widget class="QPushButton" name="r_pushButton">
       <property name="text">
        <string>Browse...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Writing path</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_w"/>
     </item>
     <item>
      <widget class="QPushButton" name="w_pushButton">
       <property name="text">
        <string>Browse...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>ANALYSE</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>SEProteinConfigStatsAppGUIClass</receiver>
   <slot>onAnalyse()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>204</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>r_pushButton</sender>
   <signal>clicked()</signal>
   <receiver>SEProteinConfigStatsAppGUIClass</receiver>
   <slot>on_r_Browse()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>287</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>w_pushButton</sender>
   <signal>clicked()</signal>
   <receiver>SEProteinConfigStatsAppGUIClass</receiver>
   <slot>on_w_Browse()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>287</x>
     <y>166</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onAnalyse()</slot>
  <slot>on_r_Browse()</slot>
  <slot>on_w_Browse()</slot>
  <slot>onGenerateModel()</slot>
 </slots>
</ui>
